ui/main/src/app/modules/cards/components/detail/detail.component.ts

```diff
@@ -247,9 +248,6 @@ export class DetailComponent implements OnChanges, OnInit, OnDestroy, AfterViewC
                         });
                 });
         }
-        this.loadTranslationForTitleAndStateName();
```

Comment:
I understand why the call to markAsReadIfNecessary is removed, but why the ones to checkIfHasAlreadyResponded, markAsReadIfNecessary and setButtonsVisibility() as well ?

Reply:
I ve seen that this methods are called in ngOnChange and each time ngOnInit is called ngOnChange is called . So it was called double .